templates/features/features_layout.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'base_layout.html' %}
+
+{% block content %}
+<section class="p-strip--main">
+  <div class="row">
+    <div class="col-2">
+      <div class="p-side-navigation is-at-the-top" id="drawer">
+        <a href="#drawer" class="p-side-navigation__toggle js-drawer-toggle u-hide--medium u-hide--large" aria-controls="drawer">
+          Toggle side navigation
+        </a>
+
+        <div class="p-side-navigation__overlay js-drawer-toggle" aria-controls="drawer"></div>
+
+        <div class="p-side-navigation__drawer">
+          <div class="p-side-navigation__drawer-header">
+            <a href="#" class="p-side-navigation__toggle--in-drawer js-drawer-toggle" aria-controls="drawer">
+              Toggle side navigation
+            </a>
+          </div>
+
+          <ul class="p-side-navigation__list">
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link{% if request.path == '/features/high-availability' %} is-active{% endif %}" href="/high-availability">High availability</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="col-10">
+
+      {% block features_content %}{% endblock features_content %}
+
+    </div>
+  </div>
+</section>
+
+{# Need help? #}
```

Comment:
I don't think we need these comment as it could go out of date and the title close by.